ssl_cert_tracker/tasks.py

```diff
@@ -41,7 +41,6 @@ def go_node(node_id, node_address):
             xml_data = nmap_task.stdout
             break
         except EnvironmentError as ex:
-            #msg = "Attempt No.: " + str(count) + " Error Scanning host:" +  node_address
```

Comment:
EnvironmentError is not a suitable exception. from the [python docs](https://docs.python.org/3/library/exceptions.html#EnvironmentError) this exception is an alias for OSError. A process may fail for many, many reasons. A generic error like OSError is not of much use.

Reply:
I change it to general exception